src/MIDebugEngine/AD7.Impl/AD7Module.cs

```diff
@@ -10,6 +10,8 @@
 using MICore;
 using System.Globalization;
 using System.Threading.Tasks;
+using System.IO;
```

Comment:
Sort using statements

Reply:
I have tried "Remove and Sort Usings" to no avail, unfortunately.